scripts/relock_dependencies.sh

```diff
@@ -41,9 +41,9 @@ set -e
 
 echo "Building Development Requirements"
 poetry lock
-poetry export -o dev-requirements.txt --without-hashes --with dev
+poetry export --format=requirements.txt --without-hashes --with dev --output dev-requirements.txt
```

Comment:
not against adding the `--format=requirements.txt` parameter, but it shouldn't make a difference since `requirements.txt` is the default option:

https://github.com/python-poetry/poetry-plugin-export?tab=readme-ov-file#available-options

Just curious about if you noticed a difference between adding it and not?

Reply:
✔️ You're right.

I missed that this was the default.